lib/modules/betteReddit.js

```diff
@@ -200,6 +205,9 @@ addModule('betteReddit', function(module, moduleID) {
 				default:
 					break;
 			}
+			if (module.options.truncateLongLinks.value) {
+				RESUtils.addCSS('overflow: hidden;text-overflow: ellipsis;white-space: nowrap;');
```

Comment:
There's a tidy new paradigm for these CSS only options: https://github.com/honestbleeps/Reddit-Enhancement-Suite/blob/master/lib/css/modules/_betteReddit.scss

Could you update to that style? Want a quick walkthrough on the pattern?